app/src/main/res/layout/list_comments_item.xml

```diff
@@ -71,6 +71,17 @@
         tools:ignore="RtlHardcoded"
         tools:text="12M" />
 
+    <ImageView
+        android:id="@+id/detail_heart_image_view"
+        android:layout_width="@dimen/video_item_detail_heart_image_size"
+        android:layout_height="@dimen/video_item_detail_heart_image_size"
+        android:layout_below="@id/itemCommentContentView"
+        android:layout_marginLeft="@dimen/video_item_detail_heart_margin"
+        android:layout_toRightOf="@+id/detail_thumbs_up_count_view"
+        android:contentDescription="@string/detail_heart_img_view_description"
+        android:visibility="gone"
+        app:srcCompat="@drawable/ic_heart" />
```

Comment:
Add `tools:visibility="visible"` here, so that the heart is visible in the Android Studio preview